vividus/src/test/java/org/vividus/configuration/VaultStoredPropertiesProcessorTests.java

```diff
@@ -156,4 +166,26 @@ void shouldUseNamespaceForEnterpriseVaultWhenConfigured() throws IOException
             ordered.verify(context).getBean(VaultTemplate.class);
         }
     }
+
+    @Test
+    void shouldPrintLogAboutDeprecation() throws IOException
+    {
+        try (var processor = new VaultStoredPropertiesProcessor(new Properties()))
+        {
+            processor.processProperty("old-property-name", "VAULT()");
```

Comment:
to reproduce comment, just try to change the test:

```suggestion
            processor.processProperty("old-property-name", "fooVAULT(secret-key)bar");
```